wikitemplate.md

```diff
@@ -202,14 +176,32 @@
 - [ ] Test that hovering the cursor over a link changes the cursor into a pointer (hand)
 - [ ] Visit https://dlptest.com/ftp-test/ and ensure that basic authentication is working by clicking on ftp://ftp.dlptest.com/ and ensuring you're prompted for credentials
 
-## Flash tests
-
-- [ ] Test that flash placeholder appears on http://www.homestarrunner.com
-- [ ] Test with flash enabled in preferences, auto play option is shown when visiting http://www.homestarrunner.com
-
-## Autofill tests
+## Ledger
 
-- [ ] Test that autofill works on http://www.roboform.com/filling-test-all-fields
+- [ ] Verify wallet is auto created after enabling payments
+- [ ] Verify monthly budget and account balance shows correct BAT and USD value
+- [ ] Click on `add funds` and click on each currency and verify it shows wallet address and QR Code
+- [ ] Verify that Brave BAT wallet address can be copied
+- [ ] Verify adding funds via any of the currencies flows into BAT Wallet after specified amount of time
+- [ ] Verify adding funds to an existing wallet with amount, adjusts the BAT value appropriately
+- [ ] Change min visit and min time in advance setting and verify if the publisher list gets updated based on new setting
+- [ ] Visit nytimes.com for a few seconds and make sure it shows up in the Payments table
+- [ ] Check that disabling payments and enabling them again does not lose state
+- [ ] Upgrade from older version
```

Comment:
We should still test that upgrading from and older version to a newer version works in terms of not losing ledger data, but I don't think we need the cases where we check for overlays/bouncing balls etc. We can change this portion to basically test if an update keeps the ledger intact when updated from a version of Brave that's 2-3 versions back. Thoughts @srirambv & @LaurenWags?